packages/@okta/vuepress-site/guides/protect-your-api/configure-packages/springboot/independ.md

```diff
@@ -1,20 +1,4 @@
-You can create a new Spring Boot project by visting the [Spring Initializr](https://start.spring.io) and selecting the dependencies **Web**, **Security**, and **Okta**, and then clicking **Generate Project** to download a zip file.
-
-You can also use the command line:
-
-```bash
-
-curl https://start.spring.io/starter.tgz  \
-  -d groupId=com.example \
-  -d artifactId=demo \
-  -d dependencies=web,security,okta \
-  -d language=java \
-  -d type=maven-project \
-  -d baseDir=demo \
-| tar -xzvf -
-```
-
-For existing projects, just include the following Apache Maven dependency: 
+Add the Otka Spring Boot SDK by including the following Apache Maven dependency: 
```

Comment:
SDK -> Starter

Reply:
Spring Boot uses the term "starter".  We also have a "starter" for the management SDK.